### ISSUE TITLE
Log error in case the Windows service does not start

### ISSUE THREAD

**Issue by svoinea** [informatics-isi-edu/iobox-win32#13]
The Windows service will start if the configuration file exist and it is valid.
If any of those condition is not satisfied, an error should be logged.

**Comment by svoinea** [comment]
The iobox log file is specified in the configuration file. If none of the above conditions is satisfied, then a default log file will be generated in the ~\Documents\iobox_service.log containing the reason the service did not start. Example:

```
2016-05-16 11:30:04,717: ERROR <serviceconfig>: Configuration file: "C:\Users\your_user_id\Documents\iobox\config\outbox.conf" does not exist.
2016-05-16 11:31:30,269: ERROR <serviceconfig>: Malformed configuration file: Expecting property name: line 56 column 8 (char 1548)
```

**Comment by svoinea** [comment]
Fixed in commit 1f655288cca110a3d47014d783fe3fd8148116aa.
